### ISSUE TITLE
Terms of service for using plugins

### ISSUE THREAD

**Issue by devcatalin** [kubeshop/monokle#1174]
Decide if we need to provide terms of service for using plugins in Monokle

**Comment by chlete** [comment]
Good point. Thanks @devcatalin. @olensmar this seems to be a legal issue, do we have somebody in the team who deals with these kind of regulations?
